source/develop/release-management/features/virt/enhance-import-export-with-ova.md

```diff
@@ -180,40 +172,32 @@ Note:
 ## Exporting a VM or Template as an OVA
 The user will be able to select a VM (or a template) to be exported as
 an OVA file. In such cases, the user needs to specify a path on a host
-where the OVA file will be created.
+or a disk on shared storage where the OVA file will be created.
 
 The following figure depicts the process of exporting a VM with a single
 disk as an OVA file:
 
 ![](../../../../images/wiki/export-ova.png)
 
-1. The front end sends the back end a VM and a path on a host.
-2. The back end asks imagio-proxy to initialize a file at the given path
-   with an estimated size of the produced OVA file.
-3. Imageio creates the OVA file.
-4. The disk is streamed from the back end to imageio-proxy, and then to
-   imageio.
-5. Imageio reads the disk and streams it into a QCOW volume within the
-   OVA.
-6. The complete OVF specification is sent from the back end to
-   imageio-proxy. imageio-proxy sends the specification to imageio, and
-   imageio streams it into the OVA file.
-
+1. The front end sends the back end a VM and an export destination.
+2. The back end creates new disk (for the ova) if the destination requires,
+   and prepares the VM source disks.
+3. The back end asks VDSM to create an OVA file with the complete OVF 
```

Comment:
you explains it afterwards - back end cannot provide the complete OVF at that stage since it doesn't know the size of the converted disks. so maybe "complete OVF" -> "complete OVF" with placeholders to be replaced later with the actual size of the disks"?